packages/core/src/components/graphs/wordcloud.ts

```diff
@@ -32,9 +32,12 @@ export class WordCloud extends Component {
 
 	render(animate = true) {
 		const self = this;
-		const svg = this.getComponentContainer()
+
```

Comment:
Add a destroy function and call it here, seems like even after destroy is added the function is never called. 
```suggestion
this.destory();
```

In the destroy function set all (click, mouseover, mouseout) event listeners to null 

```suggestion
	destroy() {
		this.parent.selectAll('text.word')
			.on('mouseover', null)
			.on('mousemove', null)
			.on('click', null)
			.on('mouseout', null);
	}
```